server/client.go

```diff
@@ -94,7 +94,7 @@ type client struct {
 	opts  clientOpts
 	start time.Time
 	nc    net.Conn
-	mpay  int
+	mpay  int64
```

Comment:
To see if this is still relevant, you could build the 32 bits (on a Linux VM) and try to run it and see if it crashes (that is, if you don't move the field)